catalog/video/video-mpegts-stream/pom.xml

```diff
@@ -219,22 +219,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.70</minimum>
+                                            <minimum>0.67</minimum>
```

Comment:
I agree that writing itests for this would be almost impossible, but I think we should try to have unit test coverage.

I can think of two options here.

We could use a template method in the Helper class for the `NetworkInterface.getNetworkInterfaces()` call, sub-class the Helper class in the unit tests class and overwrite that method to return a mock list. That should allow us to test all the scenarios that we care about. See https://github.com/codice/ddf/blob/master/platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationFileFactoryTest.java#L57-L68 for an example.

The other option would be to use PowerMock and mock the `NetworkInterface.getNetworkInterfaces()` static call.